packages/contracts-bedrock/package.json

```diff
@@ -14,10 +14,10 @@
     "bindings:ts": "nx generate @eth-optimism/contracts-ts",
     "bindings:go:no-build": "cd ../../op-bindings && make bindings-build",
     "bindings:go": "pnpm clean && pnpm build && pnpm bindings:go:no-build",
-    "prebuild": "./scripts/verify-foundry-install.sh",
+    "prebuild": "./scripts/checks/check-foundry-install.sh",
     "build": "forge build",
     "build:go-ffi": "(cd scripts/go-ffi && go build)",
-    "autogen:invariant-docs": "npx tsx scripts/invariant-doc-gen.ts",
+    "autogen:invariant-docs": "npx tsx scripts/autogen/generate-invariant-docs.ts",
     "test": "pnpm build:go-ffi && forge test",
     "test:kontrol": "./test/kontrol/scripts/run-kontrol.sh",
     "genesis": "./scripts/generate-l2-genesis.sh",
```

Comment:
Can we move this into `scripts/autogen` as well?

Reply:
I was gonna move this into a different genesis folder since making the genesis is its whole own thing